includes/admin/class-wp-job-manager-addons.php

```diff
@@ -89,9 +118,12 @@ public function output() {
 			if ( isset( $_GET['section'] ) && 'helper' === $_GET['section'] ) {
 				do_action( 'job_manager_helper_output' );
 			} else {
-				echo '<h1 class="screen-reader-text">' . __( 'WP Job Manager Add-ons', 'wp-job-manager' ) . '</h1>';
-				echo '<div id="job-manager-addons-banner" class="notice updated below-h2"><strong>' . __( 'Do you need multiple add-ons?', 'wp-job-manager' ) . '</strong> <a href="https://wpjobmanager.com/add-ons/bundle/" class="button">' . __( 'Check out the core add-on bundle &rarr;', 'wp-job-manager' ) . '</a></div>';
-				echo $addons;
+				$term = isset( $_GET['term'] ) ? sanitize_text_field( $_GET['term'] ) : null;
```

Comment:
We're adding in the ability to filter on `term` here, but we don't expose this functionality in the UI. Is this something we're planning to add later? Right now it just seems like unused code.

Reply:
👍 The end-point handles it and was going to add a filter, but might just take it out completely for now.